src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,147 @@
+statements_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtwownwrng
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    creator_user_id:
+      $ref: './fields.yaml#/user_id'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: number
+      description: |
+        Specifies the size of file uploaded by the user. (The size is represented in bytes)
+      example: 21
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the statement uploaded by the user.
+      example: 3
+      nullable: true
+    no_of_credit:
+      type: integer
+      description: |
+        Number of credit transactions in the statement uploaded by the user.
+      example: 3
+      nullable: true
+    no_of_transactions:
+      type: integer
+      description: |
+        Number of transactions in the statement uploaded by the user.
+      example: 6
+    all_statement_columns:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies headers of the statement uploaded by the user.
+      example:  
+        - account_number
+        - amount
+      nullable: false
+    start_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Start date of the statement period
+      nullable: true
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+      nullable: true
+    has_duplicate_statements:
+      type: boolean
+      description: |
+        This specifies if the statement is completely duplicate or not.
```

Comment:
p1 | This specifies if the statement is complete duplicate of already uploaded statement or not.